bin/pca-tool

```diff
@@ -419,6 +420,21 @@ def delete_image(image_id):
     return True
 
 
+def output_user_report(customer_id, assessment_id):
+    user_reports = list(UserReportDoc.objects.raw({"customer": customer_id,
+                                                   "assessment": assessment_id}).order_by([("_id", pymongo.ASCENDING)]))
+    if user_reports:
+        print("customer_id", "campaign_id", "assessment_id", "first_report", "total_num_reports")
+        for user_report in user_reports:
+            print('"{0.customer._id}", "{0.assessment._id}", "{0.campaign._id}", {0.first_report}, {0.total_num_reports}'.format(
+                user_report
+            ))
```

Comment:
The order of your field headings does not match the order of the data you are printing out (campaign and assessment IDs are reversed).